DataFormats/TauReco/src/classes_3.h

```diff
@@ -235,8 +244,19 @@ namespace DataFormats_TauReco {
 /*     edm::helpers::KeyVal<edm::RefProd<std::vector<reco::PFJet> >,edm::RefProd<std::vector<reco::PFCandidate> > > jetPFCandidateAssociation_kv; */
 /*     edm::helpers::KeyVal<edm::Ref<std::vector<reco::PFJet>,reco::PFJet,edm::refhelper::FindUsingAdvance<std::vector<reco::PFJet>,reco::PFJet> >,edm::RefVector<std::vector<reco::PFCandidate>,reco::PFCandidate,edm::refhelper::FindUsingAdvance<std::vector<reco::PFCandidate>,reco::PFCandidate> > > jetPFCandidateAssociation_kv2; */
 /*     std::map<unsigned int,edm::helpers::KeyVal<edm::Ref<std::vector<reco::PFJet>,reco::PFJet,edm::refhelper::FindUsingAdvance<std::vector<reco::PFJet>,reco::PFJet> >,edm::RefVector<std::vector<reco::PFCandidate>,reco::PFCandidate,edm::refhelper::FindUsingAdvance<std::vector<reco::PFCandidate>,reco::PFCandidate> > > > jetPFCandidateAssociation_mkv; */
-   
-
 
+    // PAT Jet associations, needed for miniAOD-based reconstruction
+    // JAN - FIXME - this should possibly go into PatCandidates
```

Comment:
I guess you mean since it refers to the pat::Jet. e had a similar discussion by the time of TauID with miniAOD input. The result (though I don't remember any more) was that the dependency remained in RecoTau,so I guess this is also fine here.

Reply:
Thanks for the clarification, will remove this comment (it was actually left to get exactly this type of comments, so I'm glad it worked ;-) )